dev/fb/datarepo-api.yaml

```diff
@@ -1,6 +1,4 @@
 ---
-image:
-  tag: 1.0.197
 env:
```

Comment:
Same here - removing this should just default to the latest jade-data-repo api version.